src/bin/internallauncher

```diff
@@ -1646,6 +1646,34 @@ class StraceDebugger(Debugger):
         launcherargs, visitargs = self.SeparateArguments(args)
         return launcherargs + strace + visitargs
 
+###############################################################################
```

Comment:
FWIW...even though this is *related* to getting stuff running on crusher, when we are looking for ways to split work over multiple PRs, something like this is a good candidate. It can be reviewed and committed wholly independently from other things.

Reply:
BTW...didn't mean to imply that is necessary now...just something to be keeping an eye out for when possible.

Reply:
Yes, that would have been a good idea.